QA/README.md

```diff
@@ -58,12 +74,13 @@ This document provides overview of "Cherrytwist" quality assurance and testing a
 
 # Environments
 - Local - setup for [Server](https://github.com/cherrytwist/Server) and [Client.Web](https://github.com/cherrytwist/Client.Web)
-- [Development](https://dev.cherrytwist.org/) - auto-deployed after each merge to development branch
-- [Test](https://test.cherrytwist.org/) - QA deploys from "github" repositories action, the required branch
-- [Acc](https://acc.odyssey.ninja:3000/ecoverse) - Project manager deploys from "github" repositories action, from master branch 
-- [Prod](https://prod.odyssey.ninja:3000/ecoverse) - Project manager deploys from "github" repositories action, from master branch
+- [Development](https://dev.cherrytwist.org/) - auto-deployed after each merge to develop branch
+- [Test](https://test.cherrytwist.org/) - QA deploys from "github" repositories action, the branch version that is being tested
+- [Acc](https://acc.odyssey.ninja:3000/) - Project manager deploys from "github" repositories action, from master branch 
```

Comment:
We should not be listing the Odyssey Acc or Prod enviornments on our Cherrytwist repo; they are a partner install